smoke-tests/apps/TelemetryProcessors/src/smokeTest/resources/applicationinsights-mask-log-body.json

```
@@ -0,0 +1,55 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100
+  },
+  "preview": {
+    "processors": [
+      {
+        "type": "log",
+        "body": {
+          "toAttributes": {
+            "rules": [
+              "^User account with userId (?<redactedUserId>[\\da-zA-Z]+)[\\w\\s]+"
```

I didn't follow what the need for the trailing portion is?

```suggestion
              "^User account with userId (?<redactedUserId>[\\da-zA-Z]+)"
```

---

this wouldn't work. there is text after user id.

The original string is "User account with userId 123456xx failed to login"

---

"User account with userId {redactedUserId} failed to login"

---

@trask i tried it, this doesn't work

---

I just tried it on your fork and it worked for me, I'll commit the suggestion and see what happens in CI

---

it worked for me too, not sure why it didn't work yesterday on my local.

---

https://github.com/MicrosoftDocs/azure-docs-pr/pull/260040